modules/gdscript/language_server/lsp.hpp

```diff
@@ -1661,7 +1661,7 @@ struct ServerCapabilities {
 		signatureHelpProvider.triggerCharacters.push_back(",");
 		signatureHelpProvider.triggerCharacters.push_back("(");
 		dict["signatureHelpProvider"] = signatureHelpProvider.to_json();
-		dict["codeLensProvider"] = false; // codeLensProvider.to_json();
+		//dict["codeLensProvider"] = codeLensProvider.to_json();
```

Comment:
If this property is not used / not meant to be passed, then the line should be removed.

But I'm not familiar with the LSP myself to say what this was meant to be / if that's something we need to actually implement.

Reply:
The LSP specification mentions that this field is optional, but doesn't give "boolean" as a valid type for it: https://microsoft.github.io/language-server-protocol/specifications/specification-current/#initialize

This is in contrast to some of the other fields, like `colorProvider`, which are optional and also explicitly allowed to be a boolean.